backend/server.js

```diff
@@ -6,12 +6,12 @@ const app = require('./app');
 require('dotenv').config();
 
 // Database
-const databaseUri = process.env.MONGODB_URI;
-mongoose.connect(databaseUri, { useNewUrlParser: true })
+const db = process.env.MONGO_URI;
```

Comment:
Please use the existing lines for defining the URI:

`const databaseUri = process.env.MONGODB_URI;
mongoose.connect(databaseUri, { useNewUrlParser: true })`

Reply:
Done